packages/rpc-provider/src/ws/index.ts

```diff
@@ -42,6 +42,13 @@ const RETRY_DELAY = 1000;
 
 const l = logger('api-ws');
 
+function eraseRecord<T> (record: Record<string, T>, cb?: (item: T) => void): void {
+  for (const key of Object.keys(record)) {
+    if (cb) cb(record[key]);
+    delete record[key];
+  }
+}
```

Comment:
```suggestion
  Object.keys(record).forEach((key): void => {
    if (cb) {
      cb(record[key]);
    }
    
    delete record[key];
  });
}
```

Prefer not using `of`